### DISCUSSION TITLE
ユーザー辞書の品詞追加提案：文語文法や方言の動詞・形容動詞

### DISCUSSION THREAD

**Discussion by ghost** [google/mozc#890] [Ideas]
ユーザー辞書で、以下の品詞を追加できるようにしていただけないでしょうか。
* 動詞上二段
  * カ行、タ行、ハ行、マ行、ヤ行、ラ行、ガ行、ダ行、バ行
* 動詞下二段
  * ア行、カ行、サ行、タ行、ナ行、ハ行、マ行、ヤ行、ラ行、ワ行、ガ行、ザ行、ダ行、バ行
* 動詞ナ変
* 動詞特殊活用（方言用）
  * 未然形、連用形、終止形…をそれぞれ別個に登録できると好い
* 名詞形動タリ活用

**Comment by hiroyuki-komatsu** [comment]
Hi tomatojuicengc5195,

Would you tell us what actual words you want to add with those part-of-speech?

**Comment by ghost** [comment]
一例ですが、
* ハ行上二段：強ふ（しふ）
* カ行下二段：設く（まうく）
* ハ行下二段：弁ふ（わきまふ）、教ふ（をしふ）、終ふ（をふ）、他多数
* マ行下二段：収む（をさむ）
* ラ行下二段；現る（あらはる）
* ナ変：去ぬ（いぬ）
* 特殊：こます
  * こませ（未然）
    * ほかはサ行四段活用に同じ。
* 名詞形動タリ：堂堂（だうだう）

といった具合です。
